lib/fast_jsonapi/object_serializer.rb

```diff
@@ -72,6 +72,8 @@ def process_options(options)
       @known_included_objects = {}
       @meta = options[:meta]
       @links = options[:links]
+      @params = options[:params] || {}
+      raise ArgumentError.new("`params` option passed to serializer must be a hash") unless @params.is_a?(Hash)
```

Comment:
A bit late to the party, but i'm curious why params has to be a hash?

Reply:
@zinosama Please review discussion starting here: https://github.com/Netflix/fast_jsonapi/pull/153#issuecomment-382211282

Reply:
Awesome. Reasonable. Thanks for the quick response 😄

Reply:
Yea no worries, sorry I did still link you to the wrong starter conversation but it seems you found the full convo!

Reply:
I opened https://github.com/Netflix/fast_jsonapi/pull/200 and meant to @ you but didn't find you in the @-list. It'd be reassuring if you could take a quick look and make sure i didn't miss anything?